### PR TITLE
Using general Source Sans Prop font family

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

**Review by phillimorland — CHANGES_REQUESTED** [review]
Looking good, there's an extra import that actions is complaining about but other then that, good to merge

File: components/TextSourceSans.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Text, StyleSheet } from 'react-native';
+import { FONTS, SIZES } from '../theme/fonts';
+
```
Comment (phillimorland) [inline components/TextSourceSans.js:4]: actions complaint about SIZES

Comment (khamui) [reply components/TextSourceSans.js:4]: ah thanks for the hint!

Out of curiosity, where do you see gh actions @phillimorland?

Comment (phillimorland) [reply components/TextSourceSans.js:4]: I was wondering if you didn't see that, so when I click on 'add your review' and check through the code in that view, actions has comments where it's sad about things, would be good if this was included in out pre commit lint though, i'll check the config

**Review by phillimorland — APPROVED** [review]

File: components/TextSourceSans.js (hunk 1 above)
Comment (khamui) [reply components/TextSourceSans.js:4]: ah yes, true..there are comments. and also true that this should be caught by our linting.

**Pull request merged**